### ISSUE TITLE
just a small suggestion

### ISSUE THREAD

**Issue by iamsoNewBee** [makamys/BucketNerf#4]
it would be better to add some vsual effect after pouring (it`s kind of stiff to just simply disappear.
and if it`s better to allow to pour near a water block?

(by the way this mod is  in harmony with regional water——water should be an limited recource XD!

**Comment by makamys** [comment]
I've actually added a particle effect when you empty a bucket, I just didn't make a release. I'll do that now.

I'm not sure what the second suggestion is supposed to mean, you should be able to pour water anywhere.

**Comment by iamsoNewBee** [comment]
i mean u can place the water block near another water block.

**Comment by tyrael93** [comment]
That looks like an issue on enviromine's side.

**Comment by iamsoNewBee** [comment]
YES,but that mod is not updated anymore. And it seems to be hard to achieve that compatibility. i think the easiest way is to just ban it.

**Comment by tyrael93** [comment]
No, I mean an issue with regular enviromine and that specific kind of tank.
Suggestion:
`don't do that`

**Comment by iamsoNewBee** [comment]
normally when other mod writers make a tank, they wouldnt consider that u what kind of water should be get from the tank because there is only one kind of water in the vanilla vision.and it could be better to cancel the same function for glass bottles if it`s possible. and if u dont like it, u can cancel this in config. emm they are just some suggestions ...

**Comment by makamys** [comment]
I added an option to cancel block interact events (the mod worked this way in 0.1.1, so it's only fair to add an option to restore the behavior).

> i mean u are able to place the water block near another water block.

This one I don't think I'll be adding, it goes a bit against the point of the mod, which is forcing you to build near natural bodies of water. And it doesn't seem like it would be trivial to implement either.

(Plus, placing water blocks next to each other one by one would be pretty annoying from a gameplay perspective, so if I did this I'd want to backport that mod that makes water source blocks spread to neighboring blocks when you break them.)
